development/external-plugins/test-untrusted/cmd/main.go

```diff
@@ -39,19 +38,34 @@ func EventHandler(server *externalplugin.Plugin, event externalplugin.Event) {
 		"pr-sender", pr.Sender.Login,
 	)
 	switch pr.Action {
-	case github.PullRequestActionOpened:
+	case github.PullRequestActionOpened, github.PullRequestActionReopened, github.PullRequestActionSynchronize, github.PullRequestActionEdited:
 		pr.GUID = event.EventGUID
 		if pr.Sender.Login == "dependabot[bot]" || pr.Sender.Login == "neighbors-dev-bot" {
```

Comment:
Do you still need to listen to `pr.Sender.Login == "neighbors-dev-bot"`? If not then maybe discard event if the sender is not `"dependabot[bot]"`
```go
if pr.Sender.Login != "dependabot[bot]" {
    return nil
}
```

Reply:
I still want to react on `neighbors-dev-bot`. It's my test user which access is limited.